amenity-points.mss

```diff
@@ -24,17 +24,16 @@
 
 /* Note that .points is also used in water-features.mss */
 .points {
-  [feature = 'tourism_alpine_hut'][zoom >= 13] {
-    point-file: url('symbols/alpinehut.p.16.png');
```

Comment:
As alpinehut.p.16.png would not be used anymore after this PR, the file should be deleted.